File: frontend/src/types/HealthProfile.ts
export interface HealthProfile {
  country: string;

  population: number;

  region: string;

  capital: string;

  cases: number;

  deaths: number;

  recovered: number;

  active: number;

  tests: number;

  recovery_rate: number;

  mortality_rate: number;

  health_index: number;
}

File: backend/app/schemas/health_profile.py
from pydantic import BaseModel

class HealthProfile(BaseModel):

    country: str

    population: int

    region: str

    capital: str

    cases: int

    deaths: int

    recovered: int

    active: int

    tests: int

    recovery_rate: float

    mortality_rate: float

    health_index: float